spec/linter-erb-spec.js

```diff
@@ -1,20 +1,38 @@
 'use babel';
 
 import * as path from 'path';
+import linter from '../lib/index';
 
-const lint = require('../lib/index.js').provideLinter().lint;
 
 describe('The ERB provider for Linter', () => {
+  const lint = linter.provideLinter().lint;
+
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
-    waitsForPromise(() => {
-      atom.packages.activatePackage('linter-erb');
-      return atom.packages.activatePackage('language-ruby').then(() =>
+
+    waitsForPromise(() =>
+      atom.packages.activatePackage('language-ruby'),
+    );
+
+    waitsForPromise(() =>
         atom.workspace.open(path.join(__dirname, 'fixtures', 'good.erb')),
-      );
-    });
+    );
+
+    atom.packages.triggerActivationHook('language-ruby:grammar-used');
```

Comment:
Without this the tests time out when waiting for `linter-erb` since there is no `language-ruby` activation